Core/AppRuntime/Source/AppRuntimeAndroid.cpp

```diff
@@ -1,9 +1,26 @@
 #include "AppRuntime.h"
+#include <exception>
+#include <sstream>
+#include <android/log.h>
 
 namespace Babylon
 {
     void AppRuntime::RunPlatformTier()
     {
         RunEnvironmentTier();
     }
+
+    void AppRuntime::DefaultUnhandledExceptionHandler(std::exception_ptr ptr)
+    {
+        try
+        {
+            std::rethrow_exception(ptr);
+        }
+        catch (const Napi::Error& error)
+        {
+            std::stringstream ss{};
+            ss << "Uncaught Error: " << error.Message() << std::endl;
+            __android_log_write(ANDROID_LOG_ERROR, "BabylonNative", ss.str().data());
```

Comment:
Should this really be the default behavior, or should we defer to `console.error`? It seems like worst case, `console.error` already does this, and best case, you are in an environment where `console.error` does something better (obviously this doesn't apply to React Native specifically, but using it as an example, it already provides a nice `console.error` experience (e.g. surfaced in the app UI, not just in debug output).

Reply:
`console.error` may not be available if the polyfill is not included. I can perhaps check, but I'm not sure that's a good idea.